### PR TITLE
Wallets

### PR DESCRIPTION
**Pull request by lthiery**
Keeping it very quick and simple for now. Adding a top-level menu items called `Wallets`. Main article summarizes the 3 different wallest: app, CLI, Ledger. 

Ledger docs from devdocs imported, but documentation for app on intercom and documentation for ledger on Github is maintained. We may want to migrate such things to this site? Anyway, keeping it short and sweet for now since legacy docs of Ledger are down.

### CODE REVIEW

File: docs/wallets/wallets/ledger.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+hide_title: true
+sidebar_label: Ledger
+slug: /ledger
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Ledger Hardware Wallet
+
+<img src={useBaseUrl("img/wallets/ledger/ledger-header.jpg")} />
```
Comment (kent-williams) [inline docs/wallets/wallets/ledger.md]: We've been leaving these out to start as Pete will likely be doing something a little different this time around.

File: docs/wallets/wallets.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+transactions, but requires more caution on behalf of its users.
+
+If you wish to proceed with using it, please follow the documentation
+[from the GitHub repository](https://github.com/helium/helium-wallet-rs).
+
+### Ledger Wallet
+
+The Helium Ledger Wallet is an application that runs on the Ledger Hardware
+wallet. The Ledger Hardware wallet is the most secure way to send and receive
+the Helium Network Token (HNT), but currently supports no other transactions.
```
Comment (kent-williams) [inline docs/wallets/wallets.md:46]: Would it make sense to add a link here to the ledger wallet guide?

**Comment by kent-williams** [comment]
Left a few comments, LGTM!

**Review by kent-williams — DISMISSED** [review]

File: docs/wallets/wallets/ledger.md (hunk 1 above)
Comment (lthiery) [reply docs/wallets/wallets/ledger.md]: Gotcha. Thanks for the heads up. I've removed it.

File: docs/wallets/wallets.md (hunk 2 above)
Comment (lthiery) [reply docs/wallets/wallets.md:46]: Yes, great catch! Thanks

**Review by PharkMillups — APPROVED** [review]
A+. Merging.

**Pull request merged**